### PR TITLE
Update README for 1.1 release

### PR DESCRIPTION
**Pull request by awh**
Merge as part of 1.1 release.

### CODE REVIEW

**Comment by awh** [comment]
Replaces #1011.

**Comment by rade** [comment]
It should be merged *as part of* the release, so the release includes the changes.

**Comment by awh** [comment]
Updated.

**Comment by awh** [comment]
I've pushed a first attempt at removing proxy/weaveDNS references; feel free to amend it directly on the branch as you see fit.

File: README.md [hunk 1]
```diff
@@ -62,17 +62,15 @@ two containers, one on each host.
...
-line are automatically attached to the weave network:
+Next we configure our environment so that containers launched via the
+docker command line are automatically attached to the weave network:
 
     host1$ eval $(weave proxy-env)
 
-Finally we run our application container; this happens via the proxy
-so it is automatically allocated an IP address and registered in DNS:
+Finally we run our application container, automatically allocating it
+an IP address and registering it in DNS:
```
Comment (rade) [inline README.md]: drop the mention of IPAM and DNS?

Comment (awh) [inline README.md]: Done.

**Pull request merged**